app/app.py

```diff
@@ -66,7 +66,11 @@ def init_handlers(self):
 
     def init_signal_handlers(self):
         signal.signal(signal.SIGINT, self.interrupt_handler)
-        signal.signal(signal.SIGQUIT, self.interrupt_handler)
+        # check for windows signal not existing
+        if getattr(signal, 'SIGQUIT', None) is not None:
```

Comment:
Slight change, let's try hasattr(signal, 'SIGQUIT').  It will return True or False.